internal/setup/setup_test.go

```diff
@@ -2240,3 +2240,53 @@ func (s *S) TestParseSliceKey(c *C) {
 		c.Assert(key, DeepEquals, test.expected)
 	}
 }
+
+var sampleText = "sample"
+
+var yamlPathTests = []struct {
+	path1, path2 *setup.YAMLPath
+	areSame      bool
```

Comment:
```suggestion
	result      bool
```

Reply:
I would name it result to avoid conflating it with some sort of equality that is not SameContent.

Reply:
Done in 1810c74.